dg/urls.py

```diff
@@ -65,8 +65,37 @@
 # loop_ivr_admin.login_template = 'social_website/login.html'
 # loop_ivr_admin.logout_template = 'social_website/home.html'
 
+from django.conf.urls import url, include
+from django.contrib.auth.models import User
+from rest_framework import routers, serializers, viewsets
+from people.models import Person
+# Serializers define the API representation.
+class UserSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = User
+        fields = ['url', 'username', 'email', 'is_staff']
+class FarmerSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
Add two lines in between classes.

Reply:
Code is outdated but same applied to other classes in the project